SimpleITK/utilities/__init__.py

```diff
@@ -49,4 +49,14 @@
 except ImportError:
     _has_vtk = False
 
+try:
+    find_spec("PySide6")
+    from .pyside import sitk2qpixmap, qpixmap2sitk
+
+    __all__.extend(["sitk2qpixmap", "qpixmap2sitk"])
```

Comment:
Please remove the code from this file. We don't want to import upside into Simpleitk.utilities for all users..

Reply:
Done.